.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,68 @@
+<!-- ↑ Provide a general summary of the issue in the Title above ↑ -->
+
+<!--
+  Hi! Thanks for filing an issue with the DITA Open Toolkit project.
+  Please take the time to answer the basic questions.
+
+  To check boxes, change the `[ ]` into `[x]` (or submit and check.)
+
+  If there is no need for certain sections, please delete those headers before
+  submitting. We know not all issues require those steps. Otherwise, please try
+  to be as detailed as possible.
+
+  If you are unsure this is a bug in DITA-OT, or if this is just a generic usage
+  question, please consider asking your question on the "dita-users" group at
+  https://groups.yahoo.com/group/dita-users.
+
+  Thanks!
+-->
+
+- [ ] I believe this to be a bug, not a question about using DITA-OT.
+- [ ] I read the [CONTRIBUTING][] file.
+
+## Expected Behavior
+<!-- If you're describing a bug, tell us what should happen. -->
+<!-- If you're suggesting a change/improvement, tell us how it should work. -->
+
+## Actual Behavior
+<!-- Tell us what happens instead of the expected behavior. -->
+
+## Possible Solution
+<!-- Optional suggestions on how to fix the issue, or implement the changes. -->
+<!-- If you know how to fix the issue, please create a pull request instead. -->
+
+## Steps to Reproduce
+<!-- Test case, Gist, set of files or steps required to reproduce the issue. -->
+
+1.
+2.
+3.
+
+<!-- Create a Gist via <https://gist.github.com/> to upload your test files. -->
+<!-- Link to the Gist from the issue or attach a .zip archive of your files. -->
+
+## Copy of the error message, log file or stack trace
+<!-- Long logs should be attached or in linked Gists, not in the issue body. -->
+
+## Environment
+<!-- Include relevant details about the environment you experienced this in. -->
+
+* DITA-OT version:
+* Operating system and version _(Linux, macOS, Windows)_:
+* How did you run DITA-OT?
+  <!-- Put an `[x]` in any of the boxes that apply: -->
+    - [ ] `dita` command
+    - [ ] Ant
+    - [ ] `startcmd`
+    - [ ] oXygen, other editor, CMS, etc.
+* Transformation type _(output format)_:
+    - [ ] eclipsehelp
```

Comment:
Since this can also be a custom transtype, maybe this should be just field to fill in?